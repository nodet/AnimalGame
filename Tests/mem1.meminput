cat
No
lobster
Does it have Antenna?
