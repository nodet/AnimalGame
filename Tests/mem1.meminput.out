Hi, let's play the animal game. You can quit at any time by typing "Quit."

Think of an animal.
Does it have Antenna? Ok. Bye!
